LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ApplicationInsightsMetricExporter.cs

```diff
@@ -58,6 +59,7 @@ internal virtual void TrackValue(Metric metric, double measurement, params strin
                 case 1: _ = metric.TrackValue(measurement, dimensions[0]); break;
                 case 2: _ = metric.TrackValue(measurement, dimensions[0], dimensions[1]); break;
                 case 3: _ = metric.TrackValue(measurement, dimensions[0], dimensions[1], dimensions[2]); break;
+                case 4: _ = metric.TrackValue(measurement, dimensions[0], dimensions[1], dimensions[2], dimensions[3]); break;
```

Comment:
What's that limitation we are talking about here? I see that the TelemetryClient supports up to 7 dimensions?

https://docs.microsoft.com/en-us/dotnet/api/microsoft.applicationinsights.metric.trackvalue?view=azure-dotnet#Microsoft_ApplicationInsights_Metric_TrackValue_System_Double_System_String_System_String_System_String_System_String_System_String_System_String_System_String_

Reply:
Yes, that's correct. However, we don't need seven dimensions, we use max three dimensions as of now.

Reply:
That's ok. However, the exception suggests that it is not supported in App Insights. Can we make that clear?

Reply:
Done in a8a99e505562fb6b46612b1a9ea4f3a0cdb34194